doc/radosgw/frontends.rst

```diff
@@ -133,10 +133,7 @@ Options
 The following is an example of the ``/etc/ceph/ceph.conf`` file with some of these options set: ::
  
  [client.rgw.gateway-node1]
- rgw_frontends = civetweb 
- request_timeout_ms = 30000 
- error_log_file = /var/log/radosgw/civetweb.error.log 
- access_log_file = /var/log/radosgw/civetweb.access.log
+ rgw_frontends = civetweb request_timeout_ms = 30000  error_log_file = /var/log/radosgw/civetweb.error.log  access_log_file = /var/log/radosgw/civetweb.access.log
```

Comment:
@cbodley can we have a more representative example in the sense that the "rgw frontends" is configured using a CSV, like
```
rgw_frontends = civetweb request_timeout_ms=30000 error_log_file=/var/log/radosgw/civetweb.error.log,beast port=7480
```

and explain to user that the all the settings for the frontends should be grouped in a single CSV as the value of "rgw_frontends" option, or this is just more confusing to enduser?

[OT] why do we discourage user from using settings like around `port = 1234` in `rgw_frontends` . i found that we complains at seeing spaces around `=`:
```c++
if (f.find("port") != string::npos) {
        // check for the most common ws problems
        if ((f.find("port=") == string::npos) ||
            (f.find("port= ") != string::npos)) {
          derr << "WARNING: radosgw frontend config found unexpected spacing around 'port' "
               << "(ensure frontend port parameter has the form 'port=80' with no spaces "
               << "before or after '=')" << dendl;
        }
      }
```
in `rgw_main.cc`.

Reply:
good points @tchaikov, thanks. we have that warning about `port` because whitespace there confuses the parser. i'll remove the extra spaces around the `=`s in this example

re: a comma-separated example, i do think that would just be confusing, and we really wouldn't recommend running multiple frontends because each one spawns a pool of 512 threads. we do spell out the formatting of `rgw_frontends` in config-ref.rst which links to this frontends.rst, so it's probably a good idea to have frontends.rst link back to config-ref.rst